index.bs

```diff
@@ -445,12 +453,14 @@ Every {{Document}} has a <dfn for="Document">post-prerendering activation steps
 
   1. Assert: |predecessorBC| is a [=top-level browsing context=].
 
-  <!-- The following are copied from the navigate algorithm, and probably could benefit from some refactoring to deduplicate. -->
+  <!-- The following except for the step that sets the |activation start time| are copied from the navigate algorithm, and probably could benefit from some refactoring to deduplicate. -->
 
   1. Cancel any preexisting but not yet [=navigate/mature=] attempts to navigate |predecessorBC|, including canceling any instances of the [=fetch=] algorithm started by those attempts. If one of those attempts has already <a lt="create and initialize a Document object">created and initialized a new `Document` object</a>, [=Document/abort=] that {{Document}} also.
 
   1. [=Prompt to unload=] the [=active document=] of |predecessorBC|. If the user [=refused to allow the document to be unloaded=], then return.
 
+  1. Let |coarsened time| be the [=coarsened shared current time=] given the |predecessorBC|'s [=active document=]'s [=relevant settings object=]'s [=environment settings object/cross-origin isolated capability=].
```

Comment:
Can a cross-origin isolated document prerender a non-cross-origin-isolated one? If so, this can create a mismatch in timer resolution. Would be better to base this on the COI capability of the prerendered document

Reply:
I see.

Changed to use `unsafe shared current time` here and `relative high resolution time` below.